### DISCUSSION TITLE
License info missing in generated client

### DISCUSSION THREAD

**Discussion by Ustice** [prisma/prisma#25316] [Q&A]
### Bug description

The [license info](https://github.com/prisma/prisma/blob/main/LICENSE) file should be included in the generated client. It's important for many organizations to have a license audit, and currently the generated client does not include this. In order for ours to be happy, we are copying over the file as a part of the build process.

### How to reproduce

1. Generate the prisma client. 
2. Check the directory, and you will not see the `LICENSE` file.

### Expected behavior

The `LICENSE` file should be included in the generated client.

### Prisma information

Any

### Environment & setup

All platforms

### Prisma Version

```
5.19.1
```

**Comment by SevInf** [comment]
Hi @Ustice, LICENSE info is included in both `prisma` and `@prisma/client` packages. Generated client is not authored by Prisma developers, it is authored by `prisma generate` command so who it will be even attributed in this case?

**Comment by ludralph** [comment]
Hi there,

To keep our discussions organized and focused on the most relevant topics, we’re reviewing and tidying up our backlog. As part of this process, we’re closing discussions that haven’t had any recent activity and appear to be outdated.

If this discussion is still important to you or unresolved, we’d love to hear from you! Feel free to reopen it or start a new one with updated details.

For more details about our priorities and vision for the future of Prisma ORM, check out our latest blog post: https://www.prisma.io/blog/prisma-orm-manifesto.

Thank you for your understanding and being part of the community!
